pkg/misc/depversion/depversion_test.go

```diff
@@ -442,3 +442,41 @@ func Test_WhichVersionMatches(t *testing.T) {
 		})
 	}
 }
+
+func Test_DoesRangeInclude(t *testing.T) {
+	testCases := []struct {
+		versions     []string
+		versionRange string
+		expect       bool
+	}{
+		{
+			versionRange: ">=1.0,<=2.0",
+			versions:     []string{"1.5"},
+			expect:       true,
+		},
+		{
+			versionRange: ">=1.0,<=2.0",
+			versions:     []string{"1.0", "2.0", "3.0"},
+			expect:       true,
+		},
+		{
+			versionRange: ">=1.0,<=2.0",
+			versions:     []string{"3.0", "2.1"},
+			expect:       false,
+		},
```

Comment:
will be good to have several test cases that check for errors as well

Reply:
just added a couple tests for errors, wasn't sure if its fine to include it in a seperate test